src/main/flight/pid.h

```diff
@@ -36,9 +36,9 @@
 #define PIDSUM_LIMIT_MIN            100
 #define PIDSUM_LIMIT_MAX            1000
 
-#define PID_GAIN_MAX 200
+#define PID_GAIN_MAX 250
 #define F_GAIN_MAX 2000
```

Comment:
```suggestion
#define F_GAIN_MAX 960
```
FF max is 960 now, DMAX doesn' t exceed 240 and PIDS can reach 250.